doc/api/dgram.md

```diff
@@ -159,7 +159,7 @@ added: v0.11.14
 -->
 
 * `options` {Object} - Required. Supports the following properties:
-  * `port` {Number} - Required.
+  * `port` {Number} - Optional.
```

Comment:
@sam-github pushed another fix. As a bonus I synced up some additional information that was missing from the options form.